test/src/test/java/org/corfudb/runtime/BackupRestoreIT.java

```diff
@@ -43,14 +43,15 @@
 @Slf4j
 public class BackupRestoreIT extends AbstractIT {
 
-    final static public int numEntries = 123;
-    final static public int valSize = 2000;
-    static final public int numTables = 5;
-    static final private String NAMESPACE = "test_namespace";
-    static final private String backupTable = "test_table";
-
-    static final private String DEFAULT_HOST = "localhost";
-    static final private int DEFAULT_PORT = 9000;
+    public static final int numEntries = 123;
+    public static final int valSize = 2000;
+    public static final int numTables = 5;
+    private static final String NAMESPACE = "test_namespace";
+    private static final String backupTable = "test_table";
+    private static final int longNameRepeat = 15;
```

Comment:
did you mean this to be 150 instead of 15?

Reply:
This repeat number means the long name will be constructed by the concatenation of `String backupTable = "test_table"` for 15 times so it's 150 chars in total.